src/test/java/bio/terra/service/snapshot/SnapshotConnectedTestUtils.java

```diff
@@ -216,4 +219,27 @@ static long queryForCount(String snapshotName, String tableName, BigQueryProject
     FieldValue countValue = row.get(0);
     return countValue.getLongValue();
   }
+
+  private static final String SNAPSHOT_DATA_TEMPLATE =
+      "SELECT * FROM `<project>.<snapshot>.<table>` ORDER BY datarepo_row_id"
+          + " LIMIT <limit> OFFSET <offset>";
+
+  static List<Map<String, Object>> getSnapshotTable(
```

Comment:
This appears to duplicate the code in `BigQueryPdao`. If it's the same code, does it need to be duplicated?

Reply:
It needs to be duplicated because otherwise BigQueryPdao needs to be initialized and `BigQueryPdao.getSnapshotTable` needs to be static, which doesn't seem ideal.

Reply:
Maybe I'm not understanding what's going on here. It looks like this code was is only called from a test, and that test only tests this code. How does that help validate that our production (aka non-test) code is correct?